BLSQ/iaso
# IA-1838: Show icon in completeness table when ou has multiple submissions

Since a previous PR, a new boolean flag (has_multiple_direct_submissions) was now returned by the API for each row in the completeness stats API

This PR shows a specific icon in the completeness table in cases this flag is set to true

Related JIRA tickets : [IA-1838](https://bluesquare.atlassian.net/browse/IA-1838)

## Self proof reading checklist


## How to test

Assign multiple submissions to a given orgunit. Make it appear in the completeness table and make sure there is a new icon (+ help text) in the "direct counts" column

## Notes

PR still in draft because I can't seem to make eslint work on my machine!

[IA-1838]: https://bluesquare.atlassian.net/browse/IA-1838?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
